Samples/Flix/FlixLibrary/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
nit: delete these?

Reply:
Argh. Meant to delete these, yes. Will remove